### ISSUE TITLE
Add S2I support for the iot-demo

### ISSUE THREAD

**Issue by ppatierno** [strimzi/strimzi-lab#2]
As proposed by @mbogoevici it could be useful having the S2I support in the iot-demo in order to build the device-app and stream-app Docker images in the OpenShift cluster directly from the GitHub repo.

**Comment by gunnarmorling** [comment]
Yes, I think that'd make sense. We could also use S2I for creating Strimzi images of [Kafka Connect with the Debezium connector](http://debezium.io/docs/openshift/) used in the lab.
